packages/react-reconciler/src/__tests__/ReactIncrementalErrorReplay-test.internal.js

```diff
@@ -11,6 +11,10 @@
 
 describe('ReactIncrementalErrorReplay-test', () => {
   it('copies all keys when stashing potentially failing work', () => {
+    // Include a renderer to ensure host config files are properly shimmed.
+    // Otherwise transient imports may cause an invariant.
+    require('react-dom');
```

Comment:
I landed another PR which needed this fix so you should be able to revert this file

Reply:
Oh, cool thanks.